dnsapi/dns_freemyip.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env sh
+#
+# Author: Recolic Keghart <root@recolic.net>
+#
+
+# acme.sh user: please set your token here!
+[ -z "$FREEMYIP_Token" ] && FREEMYIP_Token="05ddb54360621d37dea67259"
+
+################ Do not modify after this line ##############
+
+# Note: this script was executed in subshell. It means all env would be cleanup, and EXIT event would be called between different domain names. The only way to persist state is a temporary file.
+freemyip_prevdomain_tmpfile=/tmp/.acme-sh-freemyip-prevdomain
```

Comment:
dont' use temp file.
if you want to check previous domain, try to GET the txt record first.